nekoyume/Assets/_Scripts/Game/Controller/ColorController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using Nekoyume.Pattern;
+using UnityEngine;
+
+namespace Nekoyume.Game.Controller
+{
+    public class ColorController : MonoSingleton<ColorController>
```

Comment:
Instead of using `MonoSingleton<T>`, it is better to make it a static class. Because `ColorController` no need to attach to `GameObject`.
And `ColorController` does not control any colors so it is better to rename like `Palette`.

```cs
public static class Palette
{
  public static void Initialize() {}
  public static Color GetColor(int index) {}
}
```

Reply:
I fully agree :+1: